### ISSUE TITLE
Module Management should show modules even if their compatibility.minimum version is not fulfilled

### ISSUE THREAD

**Issue by gizmo2342** [foundryvtt/foundryvtt#8557]
### What happened?

During one of my last module updates I also had to fix something in the game system, which is required for the module to work properly. So I set the compatibility.minimum version accordingly.

If you now update the module but not the game system (two different actions, and believe me, there are still people out there who mix up game system and game core rules module) you will nowhere find a hint that the requirements for the module are not fulfilled.

If you enter a world where the module was not already activated before the update (not sure what happens in this case) the Module Management dialog will not show the module. Again no hint about unfulfilled requirements.

### What ways of accessing Foundry can you encounter this issue in?

- [X] Native App (Electron)
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Other

### Reproduction Steps

see above

### What core version are you reporting this for?

Version 10 (build 290)

### Bug Checklist

- [X] The issue occurs while all Modules are disabled

**Comment by aaclayton** [comment]
I read your issue, but I'm afraid I find it hard to follow. Could you explain the issue in a more clear way, perhaps with some screenshots of what you are talking about?

**Comment by gizmo2342** [comment]
My game system got an update from 1.1.0 to 1.1.1 and the module in question requires the game system with a minimal version 1.1.1

If a user installs/updates the module, but forgets to update the game system (required 1.1.1, installed 1.1.0), no compatibility warning is shown in the setup. 

In the world the Module Management dialog does not show the module at all, because it's requirements are not fulfilled. For the user it looks like "Hmm, the module is installed, but it doesn't shop up, what's wrong?" 

I would suggest to only filter modules who require a different game system. Modules with unfulfilled requirements should show up, even if not selectable, and ideally there should be a hint why they are not selectable. 

(I hope this is easier to understand.)
